apps/test_insert_deletes_consolidate.cpp

```diff
@@ -160,6 +159,7 @@ void build_incremental_index(const std::string &data_path, diskann::IndexWritePa
                                             .with_index_write_params(params)
                                             .with_search_threads(params.num_threads)
                                             .with_initial_search_list_size(params.search_list_size)
+                                            .with_num_frozen_pts(params.num_frozen_points)
                                             .with_data_type(data_type)
```

Comment:
Did you test the impact of deletes and consolidate_deletes on recall?

Reply:
I am adding code to test that as well.